README.md

```diff
@@ -81,6 +81,10 @@ Note that, assuming HTTPS is enabled, this will cause HTTP traffic to be
 redirected to port 444. You can specify both of these simultaneously as well:
 
     $ sudo snap set nextcloud ports.http=81 ports.https=444
+    
+To enable SSL (and automatic redirect from HTTP to HTTPS) using Let's Encrypt:
```

Comment:
I'm not sure this anything beyond what is [already included in the README](https://github.com/nextcloud-snap/nextcloud-snap/blob/master/README.md?plain=1#L163-L167).

Reply:
I did indeed miss the commands you referenced. Thanks for the heads up. 

Yes, your suggestion to move the CLI utilities section up near the top would be helpful. Maybe also some text in the SSL section to reference the CLI utilities section for enabling SSL? I was a bit confused by the implication that I should know how to enable SSL or have it enabled already when I was reading the HTTP/HTTPS port configuration section. I believe it would be more complete if it had at least references to all the necessary HTTP/HTTPS configs.